xml/System/Array.xml

```diff
@@ -2464,8 +2464,9 @@
   
  This method is an O(`n`) operation, where `n` is the product of all values in `lengths`.  
   
-   
-  
+[!NOTE]
+ Not all languages support arrays with non-zero lower bounds, and therefore you may not be able to cast instances of non-zero based <xref:System.Array> to the language's array type. For example, you cannot cast a 1-dimensional integer array with the lower bound 6 to C#'s `int[]` type. This results in <xref:System.InvalidCastException> during runtime with the message "Unable to cast object of type 'System.Int32[*]' to type 'System.Int32[]'.", where the asterisk (*) means non-zero based index. However, you can cast zero based arrays of any rank created with <xref:System.Array.CreateInstance(System.Type,System.Int32[],System.Int32[])> to the language's array. For example, you can cast a 2-dimensional zero based integer array created with this method to C#'s `int[,]` type.
```

Comment:
I'm not sure if `<xref:System.Array.CreateInstance(System.Type,System.Int32[],System.Int32[])>` in middle of markdowns will be interpreted as reference to the specific method. Can someone confirm?

Reply:
In addition, I also wasn't sure if line breaks in the source code will affect how this paragraph will be rendered; would it show up as one paragraph with no line breaks even if I add some line breaks in this?